Library/Homebrew/formula.rb

```diff
@@ -1457,6 +1457,7 @@ def std_cmake_args
       -DCMAKE_FIND_FRAMEWORK=LAST
       -DCMAKE_VERBOSE_MAKEFILE=ON
       -Wno-dev
+      -DBUILD_TESTING=OFF
```

Comment:
I've also sometimes seen this as
```
-DBUILD_TESTS=OFF
```
Do we know which is more common?

Reply:
`BUILD_TESTING` is the standard CTest option: https://cmake.org/cmake/help/latest/module/CTest.html